src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
>We make sure you never drop support for past frameworks. We look at your last shipped package, find out what frameworks it supported and ensure that the new package supports all of those at the same or higher API version.

That seems different from before. I understand that the package would always be supported on whatever platforms it was before. However, I thought that only meant the prior contract version, i.e. 4.0. So, for a new contract version with additional APIs, I didn't think it still had to support all platforms with the 4.1 contract version, for example.